Base/usr/share/man/man1/watch.md

```diff
@@ -0,0 +1,30 @@
+## Name
+
+watch - execute a program periodically
+
+## Synopsis
+
+`watch [-n interval] [-t] [-b] command`
+
+## Description
+
+Run a command full-screen periodically until interrupted, then return the
+aggregated error code.
+
+## Options
+
+* '-n': Interval between executions, in seconds
+* '-t': Don't print the title bar
+* '-b': Beep if the command has a non-zero exit status
+
+## Exit Values
+
+* 0 - Success
+* 1 - At least one invocation of command failed or returned non-zero
+
+## Examples
+
+```sh
+$ watch -n1 ls
+$ watch -t -- uname -a
```

Comment:
Maybe it's a task for another time, but it'd be cool if we could tell `ArgsParser` to tell `getopt` to treat all options after the first argument as arguments, not options. Then we'd be able to use that setting here in `watch`, and would not need the `--` in this example.